### ISSUE TITLE
Unable to read text, links, buttons, etc. using down arrow with JAWS

### ISSUE THREAD

**Issue by fswlee** [ithaka/pharos#364]
**Expected behavior**
Should be able to read all text, links, buttons, etc.

**Actual behavior**
User is unable to read text, links, buttons. It moves the page down instead of moving to the next item

**Steps to reproduce the issue**

1. navigate to JSTOR using JAWS
2. use the down arrow key which should read the next item (JAWS keyboard shortcuts: https://dequeuniversity.com/screenreaders/jaws-keyboard-shortcuts)
3. see that it just moves the page down

**Screenshots or code**
Here is a video of the issue

https://user-images.githubusercontent.com/5896534/174868848-178e2111-db0c-4388-987c-674dd735bebe.MOV

**Pharos version**
Indicate which version of Pharos you are using.

**Your environment**

- OS: Windows 10
- Browser: Chrome
- Version: 102.0.5005.115
- JAWS 2018

**Additional information**
This was reported to us by an external institution.

**Comment by daneah** [comment]
It's possible that more recent versions of JAWS provide stronger web component / shadow DOM support as part of the latest accessibility object model (AOM) standards. This issue comment hints at such. It would be valuable to determine whether the experience with e.g. JAWS 2021 is similar or significantly better than seen here with JAWS 2018.

**Comment by daneah** [comment]
After updates to the specific page in question, it was determined that this issue no longer appears in the general case and is unlikely to be caused directly by a Pharos component. This kind of issue can arise within or external to Pharos without care, so it's something to keep an eye toward any time focus behavior is under consideration.
